ably/realtime_client.go

```diff
@@ -44,33 +44,38 @@ func NewRealtimeClient(opts *ClientOptions) (*RealtimeClient, error) {
 	return c, nil
 }
 
-// Close
-func (c *RealtimeClient) Close() error {
-	return c.Connection.Close()
+// ConnectV12 is the same as Connection.Connect.
+func (c *Realtime) ConnectV12() {
+	c.Connection.ConnectV12()
```

Comment:
Is this still needed?

Reply:
Sorry, I got back to this after a long while and mistakenly thought I already finished with this. Should've checked. Done now at 7da5dbe129297ba6c5ca877256c538dada40f228.